src/pharmpy/tools/iivsearch/algorithms.py

```diff
@@ -42,8 +52,132 @@ def brute_force_no_of_etas(base_model, index_offset=0, keep=None):
     return Workflow(wb)
 
 
-def brute_force_block_structure(base_model, index_offset=0):
-    wb = WorkflowBuilder(name='brute_force_block_structure')
+def bu_stepwise_no_of_etas(base_model, index_offset=0):  # Should there be a "keep" argument?
+    wb = WorkflowBuilder(name='bu_stepwise_no_of_etas')
+    stepwise_task = Task(
+        "initialize_stepwise_BU",
+        initialize_stepwise_BU,
+        base_model,
+        index_offset,
+    )
+    wb.add_task(stepwise_task)
+    return wb
+
+
+def initialize_stepwise_BU(context, base_model, index_offset, base_model_entry):
```

Comment:
Is this an initialization? It's the full algorithm right?

Reply:
renamed `stepwise_BU_algorithm`